packages/blade-svelte/src/components/Amount/Amount.stories.svelte

```diff
@@ -0,0 +1,147 @@
+        options: ['xsmall', 'small', 'medium', 'large', 'xlarge', '2xlarge', '3xlarge'],
+        description: 'Size of the amount text',
+        table: {
+          defaultValue: { summary: 'medium' },
+        },
+      },
+      suffix: {
+        control: 'select',
+        options: ['decimals', 'none', 'humanize'],
+        description: 'How to display the decimal/suffix portion',
+        table: {
+          defaultValue: { summary: 'decimals' },
+        },
+      },
+      isStrikethrough: {
+        control: 'boolean',
+        description: 'Whether to show strikethrough styling',
+        table: {
+          defaultValue: { summary: 'false' },
+        },
+      },
+      isAffixSubtle: {
+        control: 'boolean',
+        description: 'Whether the currency symbol should be subtle',
+        table: {
+          defaultValue: { summary: 'true' },
+        },
+      },
+      currencyIndicator: {
+        control: 'select',
+        options: ['currency-symbol', 'currency-code'],
+        description: 'How to display the currency indicator',
+        table: {
+          defaultValue: { summary: 'currency-symbol' },
+        },
+      },
+      color: {
+        control: 'text',
+        description: 'Color token for the amount',
+      },
+    },
+  });
+</script>
+
+<script>
+  // Amount is already imported in the module context
+</script>
+
+<!-- Playground story with all controls -->
+<Story name="Playground" />
+
+<Story name="Basic Amount" args={{ value: 1234.56, currency: 'INR', type: 'body', size: 'medium' }} />
+
+<Story name="Amount Types">
+  <div style="display: flex; flex-direction: column; gap: 1rem;">
+    <div>
+      <strong>Body:</strong> <Amount value={123456.78} type="body" size="medium" currency="INR" />
+    </div>
+    <div>
+      <strong>Heading:</strong> <Amount value={123456.78} type="heading" size="large" currency="INR" />
+    </div>
+    <div>
+      <strong>Display:</strong> <Amount value={123456.78} type="display" size="xlarge" currency="INR" />
+    </div>
+  </div>
+</Story>
+
+<Story name="Amount Sizes">
+  <div style="display: flex; flex-direction: column; gap: 0.5rem;">
+    <Amount value={1234.56} type="body" size="xsmall" currency="INR" />
+    <Amount value={1234.56} type="body" size="small" currency="INR" />
+    <Amount value={1234.56} type="body" size="medium" currency="INR" />
+    <Amount value={1234.56} type="body" size="large" currency="INR" />
+  </div>
+</Story>
+
+<Story name="Different Currencies">
+  <div style="display: flex; flex-direction: column; gap: 0.5rem;">
+    <Amount value={1234.56} currency="INR" type="body" size="medium" />
+    <Amount value={1234.56} currency="USD" type="body" size="medium" />
+    <Amount value={1234.56} currency="EUR" type="body" size="medium" />
+    <Amount value={1234.56} currency="GBP" type="body" size="medium" />
+  </div>
+</Story>
+
+<Story name="Amount with Suffix">
+  <div style="display: flex; flex-direction: column; gap: 0.5rem;">
```

Comment:
Do we have utility classes for flex etc yet? If not let's add a TODO to replace these inline styles later.